README.md

```diff
@@ -119,7 +119,7 @@ Each entry in the `ipa-deployments` list define a FreeIPA cluster. All defined h
 
 **Cluster Nodes**
 
-The cluster nodes are defined for each deployment, and may have `servers` or `clients`. At least one 'server' should always be defined. If no server or client is defined, an error is returned.
+The cluster nodes are defined for each deployment, and may have `servers` or `clients`. At least one `server` should always be defined. If no server or client is defined, an error is returned.
```

Comment:
It shouldn't be a backtick as `server` is not a keyword (as are `servers` and `clients`). Maybe use double quotes (`"`)?

Reply:
I went with the double-quotes